### ISSUE TITLE
Container resource returns a non-existent IP address

### ISSUE THREAD

**Issue by efpe** [terraform-providers/terraform-provider-docker#36]
Terraform returns a non-existent IP address for a newly created container when the default bridge is disconnected as part of the `local-exec` provisioner. I have to disconnect the default bridge because of #10 

### Terraform Version
```bash
$ terraform -v
Terraform v0.11.1
+ provider.docker v0.1.1
```

### Affected Resource(s)
- docker_container
- docker_network (?)

### Terraform Configuration Files
```hcl
provider "docker" {
  host = "unix:///var/run/docker.sock"
}

data "docker_registry_image" "openldap" {
  name = "osixia/openldap:1.1.11"
}

resource "docker_image" "openldap" {
  name = "${data.docker_registry_image.openldap.name}"
  pull_triggers = ["${data.docker_registry_image.openldap.sha256_digest}"]
}

resource "docker_network" "ldap" {
  name = "ldap-network"
  driver = "bridge"
}

resource "docker_container" "ldap" {
  count = 2
  image = "${docker_image.openldap.latest}"
  name = "ldap-${count.index}"
  networks = ["${docker_network.ldap.id}"]
  hostname = "ldap-${count.index}"
  env = [
    "LDAP_LOG_LEVEL=256",
    "LDAP_REPLICATION=false"
  ]
  provisioner "local-exec" {
    // https://github.com/terraform-providers/terraform-provider-docker/issues/10
    command = "/usr/local/bin/docker network disconnect bridge ldap-${count.index}"
  }
}

output "ips" { value = "${join(", ", docker_container.ldap.*.ip_address)}" }

```

### Output

```bash
[...]
$ terraform apply
[...]
Apply complete! Resources: 4 added, 0 changed, 0 destroyed.

Outputs:

ips = 172.17.0.3,172.17.0.2

$ docker inspect ldap-0 |jq '.[0] | {networks: .NetworkSettings.Networks}'
{
  "networks": {
    "ldap-network": {
      "IPAMConfig": null,
      "Links": null,
      "Aliases": [
        "252595d82a3d"
      ],
      "NetworkID": "beeb7c11c2e8d75e078aa5b46503c42676ced6b9961f1dcd92d4de99275cc452",
      "EndpointID": "4299750f51a78d3b800bdc50d3013da3c0500fb48cd60ac7c733488caafbffb7",
      "Gateway": "172.18.0.1",
      "IPAddress": "172.18.0.3",
      "IPPrefixLen": 16,
      "IPv6Gateway": "",
      "GlobalIPv6Address": "",
      "GlobalIPv6PrefixLen": 0,
      "MacAddress": "02:42:ac:12:00:03",
      "DriverOpts": null
    }
  }
}
```

### Expected Behavior
The exported output should contain the correct IP address. I guess `NetworkSettings` is read before the provisioners run.

### Actual Behavior
Output shows the IP address of the default bridge which is disconnected by a `local-exec` provisioner.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Create the HCL posted above
2. `terraform init && terraform apply`
3. `docker inspect ldap-0 |jq '.[0] | {networks: .NetworkSettings.Networks}'`
4. `docker exec ldap-0 ip a`

### References
- #10

**Comment by melkah** [comment]
+1

**Comment by mavogel** [comment]
I'll investigate the comment https://github.com/terraform-providers/terraform-provider-docker/issues/74#issuecomment-404399428 and https://github.com/terraform-providers/terraform-provider-docker/issues/10#issuecomment-427700323 as well as the PR #92 how all those issues are related

**Comment by mavogel** [comment]
@efpe fixed with #92 and will be released with version `1.1.0` of the provider. See also https://github.com/terraform-providers/terraform-provider-docker/issues/10#issuecomment-430967364 for details. No disconnect of the default network necessary any more, because it's disconnected on the creation of the container.
